frontend/javascripts/types/schemas/datasource.schema.ts

```diff
@@ -115,6 +115,56 @@ export default {
       },
       required: ["dataFormat", "mags"],
     },
+    "types::DataLayerN5Partial": {
+      title: "DataLayerN5",
+      type: "object",
+      properties: {
+        dataFormat: {
+          const: "n5",
+        },
+        boundingBox: {
+          $ref: "#/definitions/types::BoundingBox",
+        },
+        numChannels: {
+          type: "number",
+        },
+        mags: {
+          type: "array",
+          items: {
+            type: "object",
+            properties: {
+              mag: {
+                anyOf: [
+                  {
+                    type: "number",
+                  },
+                  {
+                    $ref: "#/definitions/types::Vector3",
+                  },
+                ],
+              },
+              path: {
+                type: "string",
+              },
+              credentials: {
+                type: "object",
+                properties: {
+                  user: { type: "string" },
+                  password: { type: "string" },
+                },
+                required: ["user", "password"],
+              },
+              axisOrder: {
+                type: "object",
+                additionalProperties: { type: "number" },
+              },
```

Comment:
What does `axisOrder` look like if it is supplied?

Reply:
I think, it's something like `{x: 0, y: 1, z: 2}` (should be the same as for zarr).